### PR TITLE
Product-Specific App fixes

### PR DESCRIPTION
**Pull request by hemchander23**
Noticed that few apps involved a bit of refactoring too. Added those changes as well.

### CODE REVIEW

**Review by ravirajsubramanian — APPROVED** [review]
@hemchander23 It shows like Voucher Code Generator app is added in this PR. Check and rebase if required before merging the PR.

File: Freshworks-Samples/App-Development-Features/Serverless-Features/app-setup-and-external-events-freshdesk/manifest.json [hunk 1]
```diff
@@ -8,5 +8,8 @@
         }
       }
     }
-  }
+  },
+  "whitelisted-domains": [
+    "https://*.pipedream.net"
```
Comment (teja-kummarikuntla) [inline Freshworks-Samples/App-Development-Features/Serverless-Features/app-setup-and-external-events-freshdesk/manifest.json:13]: @hemchander23 , it's returning 404.
replace `https://*.pipedream.net` -> `https://*.m.pipedream.net`

File: Freshworks-Samples/App-Development-Features/Serverless-Features/app-setup-and-external-events-freshdesk/server/server.js [hunk 2]
```diff
@@ -1,6 +1,9 @@
 'use strict';
+/* Disclaimer: The use of mock servers in production environment is disallowed. pipedream.net can be used to debug or test your app's behavior locally.
+    However, this will not work when the app is deployed in a live account. 
+    We recommend switching to a custom mock server controlled by you in such scenarios. */
 
-const thirdPartyEndpoint = 'https://fcc15baf9a2e24ef7421ffe101394ca7.m.pipedream.net';
+const thirdPartyEndpoint = 'https://<CHANGE_THIS>.m.pipedream.net';
```
Comment (teja-kummarikuntla) [inline Freshworks-Samples/App-Development-Features/Serverless-Features/app-setup-and-external-events-freshdesk/server/server.js:6]: @hemchander23, as I'm pretty new to pipedream, It was a bit tough for me to understand how to get an endpoint for myself, would be helpful if we can extend the Readme explanation a bit more on this point alone.
Top of that, we shall convey this as a mandatory step prior to running as they have a task to replace `<CHANGE_THIS>` with the endpoint.

Comment (Saif-Shines) [reply Freshworks-Samples/App-Development-Features/Serverless-Features/app-setup-and-external-events-freshdesk/server/server.js:6]: Good point. Maybe we can link to pipedream docs if there are any from the README.md file.

**Review by Saif-Shines — APPROVED** [review]
Great work. Might have had a lot of patience!

File: Freshworks-Samples/App-Development-Features/Serverless-Features/app-setup-and-external-events-freshdesk/manifest.json (hunk 1 above)
Comment (hemchander23) [reply Freshworks-Samples/App-Development-Features/Serverless-Features/app-setup-and-external-events-freshdesk/manifest.json:13]: Good catch Teja.

File: Freshworks-Samples/App-Development-Features/Serverless-Features/app-setup-and-external-events-freshdesk/server/server.js (hunk 2 above)
Comment (hemchander23) [reply Freshworks-Samples/App-Development-Features/Serverless-Features/app-setup-and-external-events-freshdesk/server/server.js:6]: Done 👍

File: Freshworks-Samples/App-Development-Features/Serverless-Features/app-setup-and-external-events-freshdesk/README.md [hunk 3]
```diff
@@ -24,7 +24,8 @@ Observing External Events happening outside of your app will enable your app to
 ### Prerequisites
 1. Make sure you have a trial Freshdesk account created. You can always [sign up](https://freshdesk.com/signup)
 2. Ensure that you have the [Freshworks CLI](https://community.developers.freshworks.com/t/what-are-the-prerequisites-to-install-the-freshworks-cli/234) installed properly.
-3. App uses pipedream's [event sources features](https://pipedream.com/docs/event-sources/#how-do-event-sources-work) to mock an endpoint to respond with `{"success":"true"}` response. See `thirdPartyEndpoint` in `server.js`.
+3. App uses pipedream's [event sources features](https://pipedream.com/docs/event-sources/#how-do-event-sources-work) to mock an endpoint to respond with `{"success":"true"}` response. See `thirdPartyEndpoint` in `server.js`. 
+4. Ensure that the pipedream endpoint is altered accordingly in `server.js` line 6 (Replace `<CHANGE_THIS>` to your pipedrive subdomain)
```
Comment (ravirajsubramanian) [inline Freshworks-Samples/App-Development-Features/Serverless-Features/app-setup-and-external-events-freshdesk/README.md:28]: Can we not attach line numbers as they might get changed in further code refactors? The note just stands good without it, I think.

**Review by ravirajsubramanian — APPROVED** [review]
Looks good. Thanks for making these changes with patience, @hemchander23!

**Review by ravirajsubramanian — APPROVED** [review]

**Pull request merged**